### PR TITLE
dev-java/javacup: EAPI 7

### PR DESCRIPTION
**Pull request by vaukai**
Package-Manager: Portage-3.0.13, Repoman-3.0.2
Signed-off-by: Volkmar W. Pogatzki <gentoo@pogatzki.net>

### CODE REVIEW

File: dev-java/javacup/javacup-0.11b_p20151001-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+
+DEPEND="dev-java/ant-core:0
+	dev-java/jflex:0
+	>=virtual/jdk-1.8:*"
+
+RDEPEND=">=virtual/jre-1.7"
```
Comment (fordfrog) [inline dev-java/javacup/javacup-0.11b_p20151001-r1.ebuild:27]: though really not an issue, this should go to 1.8 aswell

Comment (vaukai) [reply dev-java/javacup/javacup-0.11b_p20151001-r1.ebuild]: Oops, adjusted.

**Comment by fordfrog** [comment]
@vaukai thanks!

**Comment by vaukai** [comment]
Miroslav
  
  
   
  
  
   I cannot reproduce https://bugs.gentoo.org/776655
   
  
  
   Not able to fix what I cannot reproduce, sorry.
   
  
  
   Any idea?
   
  
  
   
  
  
   Best
   
  
  
   Volkmar

**Comment by fordfrog** [comment]
@vaukai neither can i. you might try to have a look at the build.xml:53 what's there if it gives any clue.

**Comment by vaukai** [comment]
@fordfrog Could we have cc-arches for https://bugs.gentoo.org/show_bug.cgi?id=776886, https://bugs.gentoo.org/show_bug.cgi?id=776868 and https://bugs.gentoo.org/show_bug.cgi?id=776865 please

**Comment by fordfrog** [comment]
@vaukai done!

**Pull request closed without merging**